server/src/db/database.ts

```diff
@@ -166,3 +226,390 @@ export interface ShareConversationRequest {
   access_code?: string;
   expires_in_days?: number; // Optional expiration
 }
+
+export interface Category {
+  id?: number;
+  name: string;
+  description?: string;
+  image_url?: string;
+  created_at?: string;
+  updated_at?: string;
+}
+
+export interface HerbalMedicine {
```

Comment:
This is not Chinese Herbal Medicine, it's Acupoint
```suggestion
export interface Acupoints {
```